src/main/java/uk/gov/justice/hmpps/casenotes/filters/OffenderCaseNoteFilter.java

```diff
@@ -32,7 +32,7 @@ public Predicate toPredicate(Root<OffenderCaseNote> root, CriteriaQuery<?> query
             predicateBuilder.add(cb.equal(root.get("locationId"), locationId));
         }
         if (staffUsername != null) {
```

Comment:
filter needs amending to `authorUsername` too - happy to be in subsequent PR :grin:

Reply:
👍